enterprise/cloud.oracle/external/j2objc-annotations-1.3-license.txt

```diff
@@ -0,0 +1,212 @@
+Name: J2ObjC Annotations
+Description: A Java to iOS Objective-C translation tool and runtime.
```

Comment:
Wow. Why do we need this?

Reply:
This is one of the transitive dependencies of OCI libraries. I tried to remove it and fortunately it works. Removing.